flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/resource/ResourceManager.java

```diff
@@ -83,6 +83,60 @@ public ResourceManager(ReadableConfig config, MutableURLClassLoader userClassLoa
         this.userClassLoader = userClassLoader;
     }
 
+    public boolean exists(Path filePath) throws IOException {
+        boolean exist;
+        FileSystem fs = FileSystem.getUnguardedFileSystem(filePath.toUri());
+        exist = fs.exists(filePath);
+        return exist;
```

Comment:
Can we inline this?

```java
return FileSystem.getUnguardedFileSystem(filePath.toUri()).exists(filePath);
```

Reply:
Yes, this looks better. I would update it soon.